src/Core/src/Handlers/Editor/EditorHandler.Windows.cs

```diff
@@ -30,12 +30,12 @@ protected override void ConnectHandler(TextBox platformView)
 		{
 			platformView.TextChanged += OnTextChanged;
 			platformView.LostFocus += OnLostFocus;
-			platformView.Loaded += OnPlatformLoaded;
+			platformView.SizeChanged += OnPlatformViewSizeChanged;
```

Comment:
Mnnn, I am concerned about the **performance** impact. Each invalidation that impacts the size of the control will cause the `VerticalTextAlignment` and `DeleteButton` to be updated, which in both cases will look for Descendants or apply changes to the Layout, sometimes without being necessary. For example, a page with several Editors, and resizing the Window will perform the previous updates several times (without being necessary).

Reply:
> Mnnn, I am concerned about the **performance** impact. Each invalidation that impacts the size of the control will cause the `VerticalTextAlignment` and `DeleteButton` to be updated, which in both cases will look for Descendants or apply changes to the Layout, sometimes without being necessary. For example, a page with several Editors, and resizing the Window will perform the previous updates several times (without being necessary).

Hi @jsuarezruiz ,
The Loaded event does not trigger when the Editor visibility changes, which prevents the alignment from being set in this case. 
 
To address this, we used the SizeChanged event instead of the Loaded event. This approach is consistent with the method followed in the Entry control to update properties like VerticalTextAlignment and DeleteButton.
Refer to the Entry code: https://github.com/dotnet/maui/blob/main/src/Core/src/Handlers/Entry/EntryHandler.Windows.cs#L37
 
As this approach sets the respective native property, such as assigning the VerticalTextAlignment value to the native text box's VerticalAlignment property, we believe that resizing the window will only reapply the same value to the native property. It will not trigger a property change or affect any layout operations.
 
If you feel this approach is not appropriate for both Entry and Editor, we will explore other alternatives.